tests/sles4sap/saptune/mr_test_run.pm

```diff
@@ -482,11 +482,24 @@ sub wrapup_log_file {
 sub wrap_script_run {
     my ($self, $args) = @_;
     my $ret = '';
+    my $log = '/tmp/log';
 
-    $ret = script_run "$args";
+    $ret = script_run "$args > $log 2>&1";
     if ($ret) {
-        $result = 'fail';
-        record_soft_failure('Error found:jsc#TEAM-6726');
+        script_run "cat $log";
+        record_soft_failure('Error found:jsc#TEAM-7435');
```

Comment:
Do we always like to have soft failure for any ret != 0 ?

could not be the soft failure only in the else of the "if grep" ?

Reply:
> Do we always like to have soft failure for any ret != 0 ?
> 
> could not be the soft failure only in the else of the "if grep" ?

We want to continue the testing when the command's ret!=0 so a softfail is needed here. But the test case will be marked as fail ($result = 'fail';) in the end for some situations.

Reply:
FYI: https://openqa.suse.de/tests/10659520# (marked as fail in the end)

Reply:
```
$ret = script_run "$args > $log 2>&1";
if ($ret) {
    enter_cmd "cat $log";
    if (
            script_run(
"grep -E '(sysstat service.*FAIL.*exited disabled|sysstat \(cronjob\) .*FAIL.*ok|UserTasksMax.*FAIL.*max|UserTasksMax \(sapconf DropIn\).*FAIL.*regular)' $log"
            ))  {
        record_soft_failure('Error found:jsc#TEAM-7435');
        # These are known issues so mark them as 'softfail' not 'fail' for not blocking 'bot' auto approval
        # - sysstat service [FAIL] exited disabled != dead disabled
        # - sysstat (cronjob) [FAIL] ok == no link
        # - UserTasksMax [FAIL] max == 12288
        # - UserTasksMax (sapconf DropIn) [FAIL] regular == missing
    }
    else {
        $result = 'fail';
    }
```